spec/peeps_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe '#list' do
+  it 'returns a list of messages' do
+    connection = PG.connect(dbname: 'chitter_test')
+    connection.exec("INSERT INTO peeps (message, date) VALUES ('Hello', '2020-01-01');")
+    connection.exec("INSERT INTO  peeps (message, date) VALUES('Hi', '2021-01-01');")
+    connection.exec("INSERT INTO peeps (message, date) VALUES('Bonjour', '1999-01-02');")
+
+    peeps = Peeps.list
+
+    expect(peeps.length).to eq 3
+    expect(peeps.first.message).to eq 'Hello'
```

Comment:
A stronger test would be to test that the values of all peeps are returned correctly. What if a bug caused the array to contain the first tweet 3 times, for example?